libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Common/SalesforceSDKManager.m

```diff
@@ -870,6 +878,22 @@ - (void)clearClipboard
     }
 }
 
+- (void)swizzlePasteboardIfNeeded {
+    BOOL externalPasteDisabled = [SFManagedPreferences sharedPreferences].shouldDisableExternalPasteDefinedByConnectedApp;
+    if ((externalPasteDisabled && !pasteboardSwizzled) || (!externalPasteDisabled && pasteboardSwizzled)) {
+        [self swizzlePasteboard];
+    }
+}
+
+- (void)swizzlePasteboard {
+    method_exchangeImplementations(class_getClassMethod([UIPasteboard class], @selector(generalPasteboard)), class_getInstanceMethod([SalesforceSDKManager class], @selector(sdkPasteboard)));
```

Comment:
This swizzle should be run only once in the app. I think we should swizzle for sure and decide which pasteboard to use in the swizzled method. i.e. we should check the managed preference within the swizzled method. What do u think?

Reply:
So we'd keep the general pasteboard's implementation, swizzle with a new pasteboard method and then that method would either return the sdk's named pasteboard or the original general pasteboard?